### PR TITLE
BITMAKER-2209 estela-web: Connect Resources Consumption in Dashboard component

### PR DESCRIPTION
**Pull request by raymond1242**
# Description

estela-web: Connect Resources Consumption in Dashboard component

# Issue

https://tasks.bitmaker.dev/issues/2209

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by raymond1242** [comment]
@Adriana618 This PR is ready for review

**Review by Adriana618 — CHANGES_REQUESTED** [review]

File: estela-web/src/components/ProjectDashboardPage/index.tsx [hunk 1]
```diff
@@ -122,56 +138,117 @@ export class ProjectDashboardPage extends Component<RouteComponentProps<RoutePar
...
+                                                    <Text className="text-base text-estela-black-medium break-words">
+                                                        PROCESSING TIME USED
+                                                    </Text>
+                                                    {projectUseLoaded ? (
+                                                        <Text className="text-xl my-2 font-bold leading-8">
+                                                            {processingTime}
+                                                        </Text>
+                                                    ) : (
+                                                        <Spiner className="my-4" />
+                                                    )}
```
[the remark this answers is not in the document]
Comment (raymond1242) [reply estela-web/src/components/ProjectDashboardPage/index.tsx:203]: Done

**Comment by raymond1242** [comment]
@Adriana618 This PR is issues fixed

**Comment by raymond1242** [comment]
@Adriana618 This PR is ready for review

**Review by Adriana618 — APPROVED** [review]
Looks good

**Pull request merged**